test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2,7 +2,7 @@
   (:import (java.io StringReader ByteArrayInputStream ByteArrayOutputStream))
   (:require [puppetlabs.puppetserver.certificate-authority :refer :all]
             [puppetlabs.trapperkeeper.testutils.logging :as logutils]
-            [puppetlabs.certificate-authority.core :as utils]
+            [puppetlabs.ssl-utils.core :as utils]
```

Comment:
We didn't in master because we're assuming the utils were for SSL since this is the CA namespace...and I tried to make the same changes in both branches in case it might otherwise make merging more difficult